jsHelper/spicetifyWrapper.js

```diff
@@ -465,6 +465,9 @@ window.Spicetify = {
 			Routes: functionModules.find(m => m.toString().match(/\([\w$]+\)\{let\{children:[\w$]+,location:[\w$]+\}=[\w$]+/)),
 			Route: functionModules.find(m => m.toString().match(/^function [\w$]+\([\w$]+\)\{\(0,[\w$]+\.[\w$]+\)\(\!1\)\}$/)),
 			StoreProvider: functionModules.find(m => m.toString().includes("notifyNestedSubs") && m.toString().includes("serverState")),
+			Dropdown: Object.values(
+				require(Object.entries(require.m).find(([, value]) => value.toString().includes("dropDown") && value.toString().includes("isSafari"))[0])
+			).find(m => typeof m === "function"),
```

Comment:
make it a chunk like `PlaylistMenu` because i'm pretty sure it can cause problems later